### PR TITLE
Adding unit tests for FindString and ReplaceBytesValue functions

### LINKED ISSUE #174: Add unit tests to Utils package
Go thru all functions in utils package and add unit tests.
When not possible or feasible, add comments.

### PR DESCRIPTION
**Pull request by lcarrazana**
Related #174

### CODE REVIEW

**Review by eddie-knight — APPROVED** [review]

**Comment by eddie-knight** [comment]
Turns out one of my recent PRs modified a comment on the same line as this PR. Conflict needs resolved in favor of your current change here. Thanks 😅 

On another note, I noticed you linked this PR as "closing" for issue #174 - I'm not sure that's correct? Seems like we could fit some more unit tests in for the utils before closing out that task.

**Comment by lcarrazana** [comment]
> Turns out one of my recent PRs modified a comment on the same line as this PR. Conflict needs resolved in favor of your current change here. Thanks 😅
> 
> On another note, I noticed you linked this PR as "closing" for issue #174 - I'm not sure that's correct? Seems like we could fit some more unit tests in for the utils before closing out that task.

**Review by eddie-knight — APPROVED** [review]

File: internal/utils/utils.go [hunk 1]
```diff
@@ -32,7 +32,8 @@ func Int64Ptr(i int64) *int64 {
 	return &i
 }
 
-// FindString searches a []string for a specific value
+// Searches a []string for a specific value.
+// If found, returns the index of first occurrence, and True. If not found, returns -1 and False.
 func FindString(slice []string, val string) (int, bool) {
```
Comment (rparupell) [inline internal/utils/utils.go:37]: https://github.com/habibridho/go-filter-benchmark/blob/master/filter/filter.go ...
can you check above article? and see if we can do the same generic type for find.?

Comment (rparupell) [reply internal/utils/utils.go:37]: I mean i welcome your thoughts. @eddie-knight @lcarrazana

**Review by rparupell — APPROVED** [review]

**Pull request merged**